packages/core/examples/stackCanvasToWorld/index.ts

```diff
@@ -101,11 +105,16 @@ async function run() {
     ];
     // Convert canvas coordinates to world coordinates
     const worldPos = viewport.canvasToWorld(canvasPos);
+    // Convert world coordinates to image (ijk) coordinates
+    const imagePos = worldToImageCoords(viewport.getCurrentImageId(), worldPos);
 
     canvasPosElement.innerText = `canvas: (${canvasPos[0]}, ${canvasPos[1]})`;
     worldPosElement.innerText = `world: (${worldPos[0].toFixed(
       2
     )}, ${worldPos[1].toFixed(2)}, ${worldPos[2].toFixed(2)})`;
+    imagePosElement.innerText = `image (ijk): (${imagePos[0].toFixed(
+      2
+    )}, ${imagePos[1].toFixed(2)}, ${imagePos[2].toFixed(2)})`;
```

Comment:
I have a small suggestion that might help improve the readability and maintainability of the code:
```
const [i, j, k] = imagePos.map(item => item.toFixed(2));
imagePosElement.innerText = `image (ijk): (${i}, ${j}, ${k})`;
```

Reply:
I was trying the github copilot workspace to see if it works haha, i guess your suggestion makes sense